stable/heartbeat/values.yaml

```diff
@@ -58,6 +60,15 @@ resources: {}
   #  cpu: 100m
   #  memory: 100Mi
 
+priorityClassName: ""
+
+nodeSelector: {}
+
+tolerations:
```

Comment:
sorry, lets keep without any tolerations. only if there is a reason for that

Reply:
I left this for backward compatibility. I agree that no tolerations by default would be proper. I'll remove that

Reply:
removed